### PR TITLE
Add Copies of External References

### PR DESCRIPTION
**Pull request by AshtonStephens**
# Summary

1. Adds copies of external references into the SIP folder
2. Rewords signer liveness enforcement with clearer language.

### CODE REVIEW

**Comment by AshtonStephens** [comment]
Quickly changing references at the bottom to be more formal and include copies of both papers.

File: sips/sip-021/sip-021-nakamoto.md [hunk 1]
```diff
@@ -531,15 +531,18 @@ While this does induce some consolidation pressure, we believe it is the least-b
 
 # References
 
-- [1] https://eprint.iacr.org/2020/852.pdf
-- [2] https://trust-machines.github.io/wsts/wsts.pdf
+- [1] Chelsea K, Ian G (2020) FROST: Flexible Round-Optimized Schnorr Threshold Signatures. Available at https://eprint.iacr.org/2020/852.pdf and [locally][local-FROST-paper-copy] [Verified 2 February 2024]
```
Comment (obycode) [inline sips/sip-021/sip-021-nakamoto.md:534]: I find this a little strange to use first name and last initial for these authors.

Comment (AshtonStephens) [reply sips/sip-021/sip-021-nakamoto.md:534]: I just used this: https://guides.lib.monash.edu/citing-referencing/csiro-tech-rep

Comment (AshtonStephens) [reply sips/sip-021/sip-021-nakamoto.md:534]: Those are last name first, good catch!

Comment (obycode) [reply sips/sip-021/sip-021-nakamoto.md:534]: They're using "Last F", e.g. see author "Bruno Spandonide" for example.

**Review by obycode — APPROVED** [review]
LGTM. Thanks @AshtonStephens!

**Pull request merged**